functions/index.js

```diff
@@ -1,43 +1,32 @@
 const functions = require('firebase-functions');
 const Discord = require('discord.js');
-const axios = require('axios');
+const { handleDailyCC } = require("./dailyCC")
 const admin = require('firebase-admin');
-admin.initializeApp();
-const slugs = require('./constants');
+admin.initializeApp({
+  credential: admin.credential.applicationDefault()
+});
 
-exports.dailycc = functions.https.onRequest(() => {
-	const client = new Discord.Client();
+exports.dailycc = functions.https.onRequest(async (req, res) => {
+	const discordClient = new Discord.Client();
 	const { access_token } = functions.config().discord;
+	const firestore = admin.firestore();
```

Comment:
Let's move the firebase config info into its own config file so we don't have to pass it around.